src/warnet/lnnode.py

```diff
@@ -0,0 +1,55 @@
+
+import docker
+from docker.models.containers import Container
+from warnet.utils import (
+    generate_ipv4_addr
+)
+
+CONTAINER_PREFIX_LN = "ln"
+
+class LNNode:
+    def __init__(self, warnet, tank, impl):
+        self.warnet = warnet
+        self.tank = tank
+        assert impl == "lnd"
+        self.impl = impl
+        self._container = None
+        self.container_name = f"{self.tank.docker_network}_{CONTAINER_PREFIX_LN}_{self.tank.suffix}"
+        self.ipv4 = generate_ipv4_addr(self.warnet.subnet)
```

Comment:
Why don't we reuse the `tank` ipv4 address here? It will have it's own port, no?

Reply:
this is the ip address of the ln node, it lives in a different container from the tank

Reply:
oh i see what you mean, same IP different port. hmmmmmm

Reply:
My assistant says containers should all have their own IPs even if they use different ports

Reply:
Yes they very much should!